pkg/ingress/tls.go

```diff
@@ -6,17 +6,29 @@ import (
 	"strings"
 	"time"
 
+	"github.com/Sirupsen/logrus"
 	"github.com/jetstack/kube-lego/pkg/kubelego_const"
 	"github.com/jetstack/kube-lego/pkg/secret"
 	"github.com/jetstack/kube-lego/pkg/utils"
+	"github.com/prometheus/client_golang/prometheus"
 
-	"github.com/Sirupsen/logrus"
 	k8sApi "k8s.io/client-go/pkg/api/v1"
 	k8sExtensions "k8s.io/client-go/pkg/apis/extensions/v1beta1"
 )
 
 var _ kubelego.Tls = &Tls{}
 
+var (
+	certRenewalTime = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "cert_days_until_expiry",
+		Help: "Number of days remaining on ACME certificate",
+	}, []string{"domains"})
+)
```

Comment:
Please don't use global variables, place them as part of the KubeLego struct:

https://github.com/jetstack/kube-lego/blob/master/pkg/kubelego/type.go#L45

Also would call that:

```
certReqs = prometheus.NewGaugeVec(
	prometheus.CounterOpts{
		Name: "cert_expiry_time",
		Help: "Expiry time per domain",
	},
	[]string{"domain"},
)
```